fallout/benchmarks/benchmark.yaml

```diff
@@ -0,0 +1,81 @@
+ensemble:
+  server:
+    node.count: 3
+    provisioner:
+      name: ctool
+      properties:
```

Comment:
Per conversation with @adutra we likely want to explicitly locate these servers in the "performance" tenant

Reply:
These servers have now been moved to the moonshot tenant